system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -46,8 +48,11 @@ NetMonitor::NetMonitor(const rclcpp::NodeOptions & options)
   last_update_time_{0, 0, this->get_clock()->get_clock_type()},
   device_params_(
     declare_parameter<std::vector<std::string>>("devices", std::vector<std::string>())),
+  last_reasm_fails_(0),
   monitor_program_(declare_parameter<std::string>("monitor_program", "greengrass")),
-  traffic_reader_port_(declare_parameter<int>("traffic_reader_port", TRAFFIC_READER_PORT))
+  traffic_reader_port_(declare_parameter<int>("traffic_reader_port", TRAFFIC_READER_PORT)),
+  reasm_fails_check_duration_(declare_parameter<int>("ip_reasm_fails_check_duration", 1)),
```

Comment:
Please consider to change the name of variable.

Reply:
@ito-san 
I changed the word as suggested.
24a196af79dda9c44c2781252396b25fd5df32f2